### PR TITLE
Use glob for path param

### PR DESCRIPTION
**Pull request by samkio**
This was done such that multiple files
can be loaded that match given glob
patterns. Existing behaviour is preserved
for exact match.

----

You can now do things like
```yml
          path: apps/*/*/README.md
```

To load all files that match this glob pattern.

Multi line support is also included (i.e. for ignores):

```yml
          path: |
            apps/*/*/README.md
            !apps/*/infrastructure/README.md
```

The follow_symbolic_links input was also added following the recommendation here https://github.com/actions/toolkit/tree/main/packages/glob#recommended-action-inputs

Sometimes there are no matches so ignore_empty input has also been added.

### CODE REVIEW

**Comment by marocchino** [comment]
Thanks!

File: action.yml [hunk 1]
```diff
@@ -38,7 +38,15 @@ inputs:
     description: "comment message"
     required: false
   path:
-    description: "path to file containing comment message"
+    description: "glob path to file(s) containing comment message"
+    required: false
+  ignore_empty:
+    description: "Indicates whether to ignore missing or empty messages"
+    default: "true"
```
Comment (marocchino) [inline action.yml:45]: I don't know why I didn't notice this part before, but the default value is set to true. Most of the time you want to end up with an error and not skip it until you explicitly pass true. 
Also it cause of #829 and #825
We will change the default for this option in the next release. I guess you need to pass values explicitly in your CI. @samkio

Comment (samkio) [reply action.yml:45]: Ah yes it should be false sorry!

Comment (marocchino) [reply action.yml:45]: No problem. :)

**Pull request merged**